Tests/MessagingPush/PushHandling/PushEventHandlerProxyTest.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 import SharedTests
 import XCTest
 
-class PushEventHandlerProxyTest: UnitTest {
+class PushEventHandlerProxyTest: IntegrationTest {
```

Comment:
Only changed so the class had access to the new `getNewPushEventHandler()` function. Should have no effect on the tests besides that.